pageserver/src/http/openapi_spec.yml

```diff
@@ -774,15 +776,13 @@ components:
     TenantCreateRequest:
       allOf:
         - $ref: '#/components/schemas/TenantConfig'
+        - $ref: '#/components/schemas/TenantLoadRequest'
         - type: object
           required:
             - new_tenant_id
           properties:
             new_tenant_id:
               type: string
-            generation:
-              type: integer
-              description: Attachment generation number.
```

Comment:
Why does this get removed...?

Reply:
The commit adds use of TenantLoadRequest but I failed to see where would this generation come from.. Did I just miss it?

Reply:
Ugh, sorry for missing that field.